2022-02-24 21:17:10,941 INFO    MainThread:22390 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 21:17:10,941 INFO    MainThread:22390 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 21:17:10,941 INFO    MainThread:22390 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_211710-1zbaf5vp/logs/debug.log
2022-02-24 21:17:10,941 INFO    MainThread:22390 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_211710-1zbaf5vp/logs/debug-internal.log
2022-02-24 21:17:10,942 INFO    MainThread:22390 [wandb_init.py:init():404] calling init triggers
2022-02-24 21:17:10,942 INFO    MainThread:22390 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 21:17:10,942 INFO    MainThread:22390 [wandb_init.py:init():426] re-initializing run, found existing run on stack: 24zi76zf
2022-02-24 21:17:10,944 INFO    MainThread:22390 [wandb_run.py:finish():1457] finishing run simon-donike/SRCNN/24zi76zf
2022-02-24 21:17:10,945 INFO    MainThread:22390 [jupyter.py:save_history():429] not saving jupyter history
2022-02-24 21:17:10,945 INFO    MainThread:22390 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 21:17:10,945 INFO    MainThread:22390 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-02-24 21:17:10,946 INFO    MainThread:22390 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 21:17:10,947 INFO    MainThread:22390 [wandb_run.py:_restore():1752] restore
2022-02-24 21:17:11,552 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:17:11,837 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:17:12,023 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 26586
}

2022-02-24 21:17:12,128 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 26586
}

2022-02-24 21:17:12,246 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 26586
}

2022-02-24 21:17:12,351 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:12,455 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:12,558 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:12,661 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:12,764 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:12,868 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:13,405 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}

2022-02-24 21:17:13,869 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 26586
  total_bytes: 26586
}
local_info {
}

2022-02-24 21:17:15,750 INFO    MainThread:22390 [wandb_run.py:_append_files():2180] logging synced files
2022-02-24 21:17:15,758 INFO    MainThread:22390 [wandb_init.py:init():460] starting backend
2022-02-24 21:17:15,758 INFO    MainThread:22390 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 21:17:15,765 INFO    MainThread:22390 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 21:17:16,030 INFO    MainThread:22390 [backend.py:ensure_launched():222] started backend process with pid: 22592
2022-02-24 21:17:16,033 INFO    MainThread:22390 [wandb_init.py:init():469] backend started and connected
2022-02-24 21:17:16,082 INFO    MainThread:22390 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 21:17:16,083 INFO    MainThread:22390 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 21:17:16,083 INFO    MainThread:22390 [wandb_init.py:init():533] updated telemetry
2022-02-24 21:17:16,652 INFO    MainThread:22390 [wandb_init.py:init():563] communicating current version
2022-02-24 21:17:18,511 INFO    MainThread:22390 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 21:17:18,514 INFO    MainThread:22390 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 21:17:18,729 INFO    MainThread:22390 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 21:17:23,741 INFO    MainThread:22390 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 21:17:23,742 INFO    MainThread:22390 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 21:17:23,742 INFO    MainThread:22390 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 21:17:23,743 INFO    MainThread:22390 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 21:17:23,743 INFO    MainThread:22390 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 21:18:09,429 INFO    MainThread:22390 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 21:18:09,431 INFO    MainThread:22390 [wandb_run.py:_restore():1752] restore
2022-02-24 21:18:10,302 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:18:10,648 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 21:18:10,789 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:18:10,893 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:18:10,995 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25163
}

2022-02-24 21:18:11,098 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:18:11,202 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:18:11,304 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:18:11,407 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:18:11,509 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

2022-02-24 21:18:11,612 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25163
  total_bytes: 25163
}

